tests/integrationv2/processes.py

```diff
@@ -151,9 +157,13 @@ def _communicate(self, input_data=None, timeout=None):
                     # marker is found.
                     if key.fileobj is self.proc.stdin:
                         chunk = input_view[input_data_offset :
-                                           input_data_offset + _PIPE_BUF]
+                                           input_data_offset + chunk_size]
+                        # Reset chunk_size after writing a connected command
+                        chunk_size = _PIPE_BUF
                         try:
                             input_data_offset += os.write(key.fd, chunk)
+                            time.sleep(0.001)
```

Comment:
We should dive into this. A sleep indicates something isn't working as intended.

Reply:
My thoughts are that it is a bug in Openssl, but yes, I agree.